Code/Source/svFSI/heatf.cpp

```diff
@@ -323,7 +323,9 @@ void heatf_3d(ComMod& com_mod, const int eNoN, const double w, const Vector<doub
       u(2) = u(2) - N(a)*yl(6,a);
     }
   }
-
+  //if (u(2) > 0.0) {
```

Comment:
I think these three lines can be removed, right?

Reply:
removed old testing print statements in zasexton:precomputed-velocities-#202